setup.py

```diff
@@ -280,6 +280,7 @@ def finalize_options(self):
     (
         USR_LIB_EXEC + "/cloud-init",
         [
+            "tools/cloud-init-hotplugd",
```

Comment:
I'd like to avoid packaging this file on systems that do have systemd since we are already shipping systemd service/socket files with this behavior. Maybe we can conditionally append this file to data_files if we are on self.init_system != "systemd" in the `finalize_options`?

Reply:
Ok, I've changed it to be conditional on openrc.